<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="EPINHYDW0220" name="com.tam.m4.HomeTask" tests="13" failures="2" timestamp="02 Feb 2018 12:31:02 GMT" time="0.145" errors="0">
  <testcase name="testMultiplyNumbers" time="0.001" classname="com.tam.m4.HomeTask"/>
  <testcase name="testIsPositive" time="0.011" classname="com.tam.m4.HomeTask"/>
  <testcase name="testCosine" time="0.016" classname="com.tam.m4.HomeTask">
    <failure type="java.lang.AssertionError" message="expected [1.0] but found [0.0]">
      <![CDATA[java.lang.AssertionError: expected [1.0] but found [0.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at com.tam.m4.HomeTask.testCosine(HomeTask.java:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testCosine -->
  <testcase name="testMultiplyNumbers" time="0.002" classname="com.tam.m4.HomeTask"/>
  <testcase name="testMultiplication" time="0.019" classname="com.tam.m4.HomeTask"/>
  <testcase name="testSquareRootOfNegativeNumber" time="0.008" classname="com.tam.m4.HomeTask">
    <failure type="java.lang.AssertionError" message="expected [2.23606797749979] but found [5.0]">
      <![CDATA[java.lang.AssertionError: expected [2.23606797749979] but found [5.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at com.tam.m4.HomeTask.testSquareRootOfNegativeNumber(HomeTask.java:115)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testSquareRootOfNegativeNumber -->
  <testcase name="testMultiplyNumbers" time="0.005" classname="com.tam.m4.HomeTask"/>
  <testcase name="testSubtractionOfTwoNumbers" time="0.019" classname="com.tam.m4.HomeTask"/>
  <testcase name="testDivideByZero" time="0.017" classname="com.tam.m4.HomeTask"/>
  <testcase name="testDivideZeroByANumber" time="0.003" classname="com.tam.m4.HomeTask"/>
  <testcase name="testIsNegative" time="0.023" classname="com.tam.m4.HomeTask"/>
  <testcase name="testMultiplyNumbers" time="0.002" classname="com.tam.m4.HomeTask"/>
  <testcase name="testAdditionOfTwoPositiveNumbers" time="0.019" classname="com.tam.m4.HomeTask"/>
</testsuite> <!-- com.tam.m4.HomeTask -->
